packages/docs/docs/dev/getting-started.mdx

```diff
@@ -30,6 +30,14 @@ To use it, you must also install the [`fuels`](https://www.npmjs.com/package/fue
 npm install fuels @fuel-wallet/sdk
 ```
 
+To install a pre-release of `@fuel-wallet/sdk` with a specific commit, use the command:
+
+```bash
+npm install fuels @fuel-wallet/sdk@<version>-<branch>-<commit-hash>
+```
+
+Substitute `<version>`, `<branch>`, and `<commit-hash>` accordingly. Check the [npm version history](https://www.npmjs.com/package/@fuel-wallet/sdk?activeTab=versions) for the exact identifiers.
```

Comment:
We can use the @master tag to get the latest version from master. Maybe it would be more straightforward.